packages/wonder-blocks-cell/src/components/detail-cell.tsx

```diff
@@ -94,7 +94,7 @@ const DetailCell = function (props: DetailCellProps): React.ReactElement {
 
 const styles = StyleSheet.create({
     subtitle: {
-        color: color.offBlack64,
+        color: "#6D6F74",
```

Comment:
Hardcoding for now based on the thread in WB-1797. It still has a11y color warnings so will get feedback!